docs-en/chapter_data_structure/character_encoding.md

```diff
@@ -47,41 +47,41 @@ The encoding rules for UTF-8 are not complex and can be divided into two cases:
 
```

Comment:
@@ -47,41 +47,41 @@ The encoding rules for UTF-8 are not complex and can be divided into two cases:

became missing in the newly changed version, im not sure if this will be auto added, but just saying.